src/Diagnostics.DataProviders/KustoSDKClient.cs

```diff
@@ -15,48 +14,61 @@
 using Diagnostics.ModelsAndUtils.Models;
 using Newtonsoft.Json;
 using Diagnostics.DataProviders.Exceptions;
-using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace Diagnostics.DataProviders
 {
     class KustoSDKClient : IKustoClient
     {
+        private readonly KustoDataProviderConfiguration _config;
         private string _requestId;
         private string _kustoApiQueryEndpoint;
         private string _appKey;
         private string _clientId;
         private string _aadAuthority;
         private static ConcurrentDictionary<Tuple<string, string>, ICslQueryProvider> QueryProviderMapping;
+        private static List<string> exceptionsToRetryFor = new List<string>();
 
         /// <summary>
         /// Failover cluster mapping
         /// </summary>
-        public ConcurrentDictionary<string, string> FailoverClusterMapping { get; set; }
+        private ConcurrentDictionary<string, string> FailoverClusterMapping { get; set; }
```

Comment:
>private [](start = 8, length = 7)

Is this property not used anywhere else in the project?

Reply:
Yup. This shouldnt be public